lib/automations/release/branch-create-handler.js

```diff
@@ -116,6 +117,7 @@ const branchHandler = async ( context, octokit, config ) => {
 		);
 		changelog = await getChangelog( changelogItems, config );
 		devNoteItems = getDevNoteItems( changelogItems, config );
+		await attachChangelogToRelease( context, octokit, changelog );
```

Comment:
The GitHub release won't have been created yet by this point right? Won't this just attach this to the previous release?